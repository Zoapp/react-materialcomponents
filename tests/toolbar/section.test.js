import React from "react";
import renderer from "react-test-renderer";
import Section from "libs/toolbar/section";

describe("toolbar/Section", () => {
  it("can render children", () => {
    const component = renderer.create(<Section>Children are here!</Section>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("can be aligned", () => {
    const component = renderer.create(<Section align="start" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("can return True if text automatically shrinks to fit in the available column width", () => {
    const component = renderer.create(<Section shrinkToFit />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
