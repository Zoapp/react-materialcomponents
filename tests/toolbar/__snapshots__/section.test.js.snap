// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toolbar/Section can be aligned 1`] = `
<section
  className="mdc-toolbar__section mdc-toolbar__section--align-start"
/>
`;

exports[`toolbar/Section can render children 1`] = `
<section
  className="mdc-toolbar__section"
>
  Children are here!
</section>
`;

exports[`toolbar/Section can return True if text automatically shrinks to fit in the available column width 1`] = `
<section
  className="mdc-toolbar__section mdc-toolbar__section--shrink-to-fit"
/>
`;
